app/data_migrations/change_enrollment_details.rb

```diff
@@ -101,7 +101,9 @@ def revert_termination(enrollments)
 
   def terminate_enrollment(enrollments)
     terminated_on = Date.strptime(ENV['terminated_on'].to_s, "%m/%d/%Y")
+    raise "Not a valid termination date.  Please format date as mm/dd/yyyy." if terminated_on.year < 1000 && !Rails.env.test?
```

Comment:
Might want to consider using a regex to validate the format here.  That way we can enforce the format of the entire date string (not just the year) and also make the error message correlate to the code logic a bit more closely.